doc/ForcedUpdate.md

```diff
@@ -0,0 +1,22 @@
+# Forced Update
+
+To force an update, we suscribe to the UpdateRequired Event from the UpdateRequiredservice in the coreStartup.
```

Comment:
Explain how the update is forced, because the user actually has to do the update himself. It's not automatic and I think it's important to explain what is the user experience.